onert-micro/luci-interpreter/src/kernels/UnidirectionalSequenceLSTM.cpp

```diff
@@ -0,0 +1,884 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kernels/UnidirectionalSequenceLSTM.h"
+#include "kernels/Utils.h"
+
+#include "PALApplyActivationToVector.h"
+#include <tensorflow/lite/kernels/internal/tensor_utils_common.h>
+#include <tensorflow/lite/kernels/internal/reference/portable_tensor_utils.h>
+namespace luci_interpreter
+{
+namespace kernels
+{
+namespace lstm
+{
+namespace
+{
+
+using namespace tflite;
+
+void UpdateLstmCellFloat(int n_batch, int n_cell, float *cell_state, const float *input_gate,
```

Comment:
Optional (maybe for future refactoring): How about all functions from this `lstm` namespace move to another helper file (smth like FloatLSTMEvalKernel), due to it is very difficult to read this file, and now it is only for float and when we will add integer (and maybe hybrid part) it will be very huge file.
But now it is okay leave it like this, but add todo note please

Reply:
Or maybe create PAL for mcu and move these functions to this file.